Source/Csla.Shared/Reflection/ServiceProviderMethodCaller.cs

```diff
@@ -236,7 +265,12 @@ public static ServiceProviderMethodInfo FindDataPortalMethod<T>(Type targetType,
             throw new TargetParameterCountException(cacheKey);
           else
           {
+#if NET5_0_OR_GREATER
```

Comment:
Isn't this #if block invoking the same line of code either way?

Reply:
Same here as in previous place.